e2e/go.mod

```diff
@@ -3,45 +3,45 @@ module github.com/openshift/cluster-capi-operator/e2e
 go 1.18
 
 require (
-	github.com/aws/aws-sdk-go v1.44.77
-	github.com/google/go-cmp v0.5.8
-	github.com/onsi/ginkgo/v2 v2.3.1
-	github.com/onsi/gomega v1.22.1
+	github.com/aws/aws-sdk-go v1.44.190
+	github.com/google/go-cmp v0.5.9
+	github.com/onsi/ginkgo/v2 v2.7.0
+	github.com/onsi/gomega v1.26.0
 	github.com/openshift/api v0.0.0-20220921125526-1866ef90edbf
-	k8s.io/api v0.25.1
-	k8s.io/apimachinery v0.25.1
-	k8s.io/client-go v0.25.1
-	k8s.io/utils v0.0.0-20220728103510-ee6ede2d64ed
-	sigs.k8s.io/cluster-api v1.2.2
-	sigs.k8s.io/cluster-api-provider-aws v1.3.0
-	sigs.k8s.io/cluster-api-provider-gcp v1.1.0
+	k8s.io/api v0.25.4
+	k8s.io/apimachinery v0.25.4
+	k8s.io/client-go v0.25.4
```

Comment:
We can't bump to 1.26 because upstream doesn't support it yet right?

Reply:
Correct.

Reply:
I think there is a longer conversation around CAPI versioning involved here. I'd like to hold merging this until we have a team chat about it.